DigitalLearningSolutions.Data.Tests/DigitalLearningSolutions.Data.Tests.csproj

```diff
@@ -21,6 +21,7 @@
     <PackageReference Include="FluentMigrator" Version="3.2.9" />
     <PackageReference Include="FluentMigrator.Runner" Version="3.2.9" />
     <PackageReference Include="FluentMigrator.Runner.SqlServer" Version="3.2.9" />
+    <PackageReference Include="JetBrains.DotMemoryUnit" Version="3.1.20200127.214830" />

```

Comment:
Is this added on purpose/is it related to the ticket?

Reply:
Nope, miss click when running a unit test, didn't realise it had added a package